packages/instrumentation-typeorm/src/types.ts

```diff
@@ -11,4 +11,6 @@ export interface TypeormInstrumentationConfig extends InstrumentationConfig {
      * and value of the module version.
      */
     moduleVersionAttributeName?: string;
+    /** Some methods such as `getManyAndCount` execute multiple queries. to instrument those set this to `true` */
+    enableInternalInstrumentation?: boolean;
```

Comment:
should use suppressInternalInstrumentation instead
https://github.com/aspecto-io/opentelemetry-ext-js/issues/111